.github/workflows/ci.yml

```diff
@@ -23,7 +23,6 @@ jobs:
     with:
       envs: |
         - linux: check-style
-        - linux: build-dist
```

Comment:
Now that I look at it, pre-commit is doing the check-style too, so the whole `check` block can be removed.